packages/ag-charts-website/src/content/docs/charts-waterfall-series/_examples/basic-waterfall-horizontal/main.ts

```diff
@@ -12,7 +12,8 @@ const options: AgChartOptions = {
     },
     series: [
         {
-            type: 'waterfall-column',
```

Comment:
Was this example meant to be a vertical column? now it's a  waterfall-bar

Reply:
I think `git mv` got a bit confused here - the direction configured with the series matches the example name, so I think it's correct,